.env.example

```diff
@@ -0,0 +1,3 @@
+AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
Is this key valid? This is a public repo, we should revoke this key if it's a valid one.

Reply:
There end with "EXAMPLE", so I don't think so

Reply:
Haven't noticed that, so guess it's ok but we could keep it empty